addons/website_sale/static/src/xml/website_sale.editor.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
```

Comment:
Review the indentation of all your files. We ****never**** use tabs, always spaces. Also do not indent the first level (t-name)